students/BrianNisonger/break_me.py

```diff
@@ -0,0 +1,18 @@
+def NameError():
+	print (x)
+
+def TypeError():
+	a=5;
+	b="17"
+	return a+b
+#def SyntaxError():
+	#for(i;i++;i=0)
```

Comment:
funny how you can't write C and expect a Python interpreter to understand it :-)